docs/UserGuide.md

```diff
@@ -406,10 +421,52 @@ Duplicate handling:
 
 ### Address
 
+Format: String of any value.
+
+Constraints:
+* The first character of the string cannot be a whitespace.
+
+Duplicate handling:
+* Any number of students can have the same address.
+
+> [!Note]
+> Constraint rationale: A whitespace cannot be the first character of the string, such that " " cannot be a valid input.
```

Comment:
Is this a format constraint or simply a bug, and if so, should we fix it?

Reply:
It was in the original AB3, so I think it is a format constraint